prost-build/Cargo.toml

```diff
@@ -12,7 +12,7 @@ documentation = "https://docs.rs/prost-build"
 readme = "README.md"
 description = "A Protocol Buffers implementation for the Rust Language."
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.70"
```

Comment:
There are actually more than one `rust-version` definitions in the code base. Can you update the others as well?